### DISCUSSION TITLE
Tabs at bottom of card

### DISCUSSION THREAD

**Discussion by Jawyman42** [kinghat/tabbed-card#51] [Feature Requests]
For mobile dashboards it would be a great feature to be able to put the tabs at the bottom of the card or put them as a side scroller or side tabs.

**Comment by kinghat** [comment]
the material guidelines dont recommend putting tabs at the bottom as a ux pattern but i will probably implement it. sides not so much. thanks for the request.

**Reply by ildar170975** [reply]
Managed to do it by card-mod:
```
type: custom:mod-card
card_mod:
  style:
    tabbed-card $:
      mwc-tab:
        $:
          mwc-tab-indicator $: |
            .mdc-tab-indicator__content--underline {
               align-self: flex-start !important;
            }
    .: |
      tabbed-card {
        display: flex;
        flex-direction: column-reverse;
      }
card:
  type: custom:tabbed-card
  attributes:
    icon: mdi:circle
  tabs:
    - card:
        <<: &ref_card
          type: entities
          entities:
            - sun.sun
      attributes:
        label: label 1
    - card:
        <<: *ref_card
      attributes:
        label: label 2
    - card:
        <<: *ref_card
      attributes:
        label: label 3
```
![image](https://user-images.githubusercontent.com/71872483/219113528-857ece72-aba1-4d01-a1ab-883bd31de822.png)
[Check here](https://community.home-assistant.io/t/card-mod-add-css-styles-to-any-lovelace-card/120744/4280?u=ildar_gabdullin)

**Reply by Californian** [reply]
That's my main point of disagreement with MD: tabs at the top are nearly unusable on medium to large phones, especially without the ability to swipe between them (which isn't always possible depending on the contents of the tabs).

**Reply by Californian** [reply]
Have you written any css to get the labels to float to the bottom/get the tab scrollable instead of the group with the tab?

**Reply by ildar170975** [reply]
> get the labels to float to the bottom/get the tab scrollable instead of the group with the tab?

Unclear, please elaborate & attach some pics
